### ISSUE TITLE
imod.wq.bas thickness() method doesn't guarantee correct dimension order

### ISSUE THREAD

**Issue by Manangka** [Deltares/imod-python#302]
In GitLab by @Huite on Dec 15, 2022, 16:37

Thickness is currently computed as:

```python
        th = xr.concat(
            [
                self["top"] - self["bottom"].sel(layer=1),
                -1.0 * self["bottom"].diff("layer"),
            ],
            dim="layer",
        )

```

This does not guarantee the right dimension order if top is a constant, and bottom is 3D, as it will give dimension order (y, x, layer).

**Comment by Manangka** [comment]
In GitLab by @Huite on Dec 15, 2022, 16:51

Inform Gijs once this has been fixed.

**Comment by Manangka** [comment]
In GitLab by @JoerivanEngelen on May 2, 2023, 14:07

Fixed in release 0.13.0
